packages/mdc-menu/foundation.js

```diff
@@ -564,6 +597,38 @@ class MDCMenuFoundation extends MDCFoundation {
   isOpen() {
     return this.isOpen_;
   }
+
+  /** @return {?Element} */
```

Comment:
I think we try NOT to reference HTML Element in our Foundation classes. Maybe this public method is not necessary...